vignettes/GOvis.Rmd

```diff
@@ -0,0 +1,497 @@
+---
+title: "Visualizing and Distances Using GO"
+authors:
+  - name: "R. Gentleman"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.time(), '%B %d , %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Visualizing and Distances Using GO}
+  %\VignetteDepends{Biobase,genefilter,graph,GO.db,hgu95av2.db,xtable,multtest}
+  %\VignetteSuggests{Rgraphviz}
+  %\VignetteKeywords{EDA, graphs, ontology, visualization}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignettePackage{GOstats}
+output:
+  BiocStyle::html_document
+bibliography: GOstats.bib
+link-citations: yes
+---
+
+# Introduction {#introduction .unnumbered}
+
+The basic characteristics of the GO [@GO] data are described in @GO and the
+interested reader is referred there for more details. In this paper we assume
+that readers are familiar with the basic DAG structure of GO and with the
+mappings of genes to GO terms that are provide by GOA [@GOA].
+We examine  some of the different visualization problems and data
+analytic problems that can be addressed using these data. To
+make some of the concepts concrete and to provide extensive, but related
+examples we will make use of a microarray data set [@Chiaretti2004].
+
+We begin with brief descriptions and reminders of some of the important
+concepts, as well as of our example data. Once that has been done we will
+then consider several different bioinformatic tasks that are of interest
+and show how these tasks can be carried out using software from the Bioconductor
+Project.
+
+```{r Setup, echo=FALSE, results='hide', message=FALSE}
+library("Biobase")
+library("annotate")
+library("GOstats")
+library("xtable")
+library("multtest")
+library("Rgraphviz")
+library("hgu95av2.db")
+library("GO.db")
+library("genefilter")
+library("ALL")
+```
+
+The results reported in this document are based on
+`r packageDescription("hgu95av2.db")[["Version"]]` version of the
+`r Biocpkg("hgu95av2.db")` data package and on
+`r packageDescription("GO.db")[["Version"]]` of the `r Biocpkg("GO.db")` package.
```

Comment:
Use BiocStyle's `pkg_ver()` to get the version of packages. Can I also ask you to format this paragraph nicely?

Reply:
Ohk I will do that do I need to do any more changes?

Reply:
I'm in the middle of reviewing. I will send a message when the review is complete.